sdk/openai/openai/src/models/audio.ts

```diff
@@ -100,6 +82,20 @@ export interface GetAudioTranslationOptions extends OperationOptions {
   model?: string;
 }
 
+/** The result format of an audio task */
+export type AudioResultFormat =
+  | "json"
```

Comment:
The docstring for json was dropped

Reply:
Not entirely sure why, I'll raise an issue for this